ffi/examples/read-table/read_table.c

```diff
@@ -150,13 +151,67 @@ void free_partition_list(PartitionList* list) {
   free(list);
 }
 
+static const char *LEVEL_STRING[] = {
+  "ERROR", "WARN", "INFO", "DEBUG", "TRACE"
+};
+
+#define RED   "\x1b[31m"
```

Comment:
Maybe worth a quick comment identifying these as ANSI color escape sequences?
(I don't think that's a super well-known blast from the past)